server/JWT.js

```diff
@@ -27,7 +27,7 @@ let JWT = {
       var parts = req.headers.authorization.split(' ')
       if (parts.length === 2 && /^Bearer$/i.test(parts[0])) { return parts[1] }
       throw new Error('Format is Authorization: Bearer [token]')
-    } else if (req.param('token')) {
+    } else if (req.query.hasOwnProperty('token') || req.body.hasOwnProperty('token')) {
```

Comment:
how is this different from `req.param` which checks body and query?

Reply:
`req.param` can return null, undefined, empty string, etc and the IF statement won't pass

Reply:
got it, use req.allParams and a _.has